include/gadget/user_stack_map.h

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2025 The Inspektor Gadget authors
+
+#ifndef __USER_STACK_MAP_H
+#define __USER_STACK_MAP_H
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+#include <gadget/types.h>
+
+#define USER_MAX_STACK_DEPTH 127
+#define USER_STACK_MAP_MAX_ENTRIES 10000
+
+struct {
+	__uint(type, BPF_MAP_TYPE_STACK_TRACE);
+	__uint(key_size, sizeof(u32));
+	__uint(value_size, USER_MAX_STACK_DEPTH * sizeof(u64));
+	__uint(max_entries, USER_STACK_MAP_MAX_ENTRIES);
+} ig_ustack SEC(".maps");
+
+struct inode___with_timespec64 {
+	struct timespec64 i_mtime;
```

Comment:
Some kernels define it like this:
```
struct timespec64	__i_mtime;
```
https://github.com/torvalds/linux/blob/v6.8/include/linux/fs.h#L677

We need to support this too.